ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -384,6 +384,10 @@ StatusSectionLayout {
                 communityTokensStore.computeSelfDestructFee(
                     remotelyDestructTokensList, tokenKey)
 
+            onRemotelyDestructCollectiblesOpened:
```

Comment:
Same comment, not necessary a new signal, since we just want to
`communityTokensStore.computeSelfDestructFee` but with `accountAddress` parameter.

You will need to also add the parameter into:
1. `CommunityTokensStore/computeSelfDestructFee` --> A task for updating backend to take into account address value will be needed
2. `CommunityTokensStore/remoteSelfDestructCollectibles` -->  A task for updating backend to take into account address value will be needed

@endulab this pr is updating remotely destruct popup to show fees box and we will need to update backend in order to take into account **account address** while **computing fee** and doing **remotely destruct action.**

@alexandraB99 you should review also the last remotely desctruct action in the flow since it also will need to pass now to the store the `accountAddress` value.